packages/shared/styles/variables/_colors.scss

```diff
@@ -34,5 +33,32 @@ $c-bg-primary: $c-light-primary !default;
 $c-bg-secondary: $c-light-secondary !default;
 
 // Border
-
 $c-border: $c-light-primary;
+
+// Theme colors
+$primary: $c-green-primary !default;
```

Comment:
maybe `accent-primary` and secondary since this refers to page accent?

Reply:
To be honest I think it's redundant, imo we don't have to be so descriptive on these variables names, and we may have variables derivative from `$primari` and `$secondary`, `accent-primary` it should generate big variables names.
Wdyt? I would like to hear more opinions color variables.

Reply:
Good point, I'm not so tied to it. Only please use c- prefix for colors